app/views/admin/delivery_partners/users/delete.html.erb

```diff
@@ -1,9 +1,9 @@
 <% content_for :title, "Delete user" %>
 <h1>Do you want to delete this user?</h1>
-<p class="govuk-!-font-weight-bold">Supplier user: <span class="govuk-!-font-weight-regular"><%= @user.full_name %></span></p>
```

Comment:
I think we are missing the summary error message box in this form

Reply:
@cwrw 
We're not rendering this page to show the errors. We're deleting and then redirecting to index page